### PR TITLE
Week 19 -  Karolin Albinsson - project-happy-thoughts-api

### PR DESCRIPTION
**Pull request by karolinalbinsson**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,29 +1,97 @@
...
+
+//get 20 happythoughts
+app.get("/thoughts", async (req, res) => {
+	try {
+		const thoughts = await Thought.find().sort({ createdAt: -1 }).limit(20);
+		res.status(200).json(thoughts);
+	} catch (err) {
+		res
+			.status(400)
+			.json({ message: "Could not like thought.", errors: err.errors });
```
Comment (Andrea-Osmar) [inline server.js:90]: Shouldn't the message here be about the thought instead of like?

File: README.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+I started out creating the endpoints one by one and tested them out in postman. When I was happy with the result, I connected it in my happy-thoughts frontend. 
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+Frontend repo and link: 
+https://github.com/karolinalbinsson/happy-thougths-own-api
```
Comment (Andrea-Osmar) [inline README.md:12]: maybe you should include the deployed backend here as well.

**Review by Andrea-Osmar — APPROVED** [review]
Great yob Karolin! Your page looks beautiful and functions just perfect! 
I really like the design of your frontend and all you're code is clean and easy to follow. You are consistent with indentation and the use of semicolons. 
very good job this week! You should definitely be proud of yourself 🥇

File: server.js [hunk 3]
```diff
@@ -1,29 +1,97 @@
...
+app.get("/", (req, res) => {
+	res.send("Hello world");
+});
+
+//Create new thought
+app.post("/thoughts", async (req, res) => {
+	try {
+		const thought = new Thought({ message: req.body.message });
+		await thought.save();
+		res.status(200).json(thought);
```
Comment (KarolCislowski) [inline server.js:55]: this status should be 201 (CREATED)

**Review by KarolCislowski — APPROVED** [review]
Great project 👍🏻

**Pull request closed without merging**